filebeat/module/nats/log/_meta/fields.yml

```diff
@@ -4,3 +4,53 @@
     Nats log files
   release: beta
   fields:
+    - name: client
+      type: group
+      description: >
+        Fields from NATS logs client.
+      fields:
+        - name: id
+          type: integer
+          description: >
+            The id of the client
+    - name: msg
+      type: group
+      description: >
+        Fields from NATS logs message.
+      fields:
+        - name: bytes
+          type: long
+          description: >
+            Size of the payload in bytes
+        - name: direction
+          type: text
```

Comment:
Shouldn't this be `keyword`. Does this map to `network.direction` from ECS? https://github.com/elastic/ecs

Reply:
yes it can be mapped. That's why I performed the appropriate change

Reply:
But please change the datatype to `keyword`.

Reply:
I believe that I have removed all the unwanted field definitions but there may be something it slipped. Did you spot any leftover amongst them? Please let me know to remove it as well. :)